honestcash-v2/src/app/auth/store/auth.effects.spec.ts

```diff
@@ -236,17 +210,6 @@ describe('auth.effects', () => {
       });
     });
     describe('SignUpSuccess', () => {
-      it('should correctly return UserSetup with and password', () => {
```

Comment:
these tests are removed because on SignUpSuccess it is no longer the case where we setup the user or the wallet